app/assets/stylesheets/application.css.scss

```diff
@@ -23,17 +23,35 @@ body {
 
 .application-body{
   background-image: url("http://i.imgur.com/9olXuLe.jpg");
-  opacity: 0.97;
 }
 
 .application-body-doom{
   background-image: url('doom.jpeg');
-  opacity: 0.97;
   background-size:     cover;
   background-repeat:   no-repeat;
   background-position: center center;
 }
 
+.application-body-shire{
+  background-image: url('shire.jpeg');
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center;
+}
+
+.application-body-bree{
+  background-color: black;
+}
+
+
+td, th {
+    padding: 0px 0px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered border-radius, display, padding, text-align, vertical-align<br>Shorthand form for property `padding` should be written more concisely as `0px` instead of `0px 0px`<br>`0px` should be written without units as `0`